CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 2.8.2)
```

Comment:
This essentially drops the support for RHEL 6, because only Cmake 2.8.12 is available on that distro.. Can you go back to 2.8 ?

Reply:
Fixed, turns out 2.8.2 was just for a sample, actual minimum is 2.6.4